integration/consume_integration_test.go

```diff
@@ -225,6 +230,19 @@ func (ti *testConsumeIntegration) test(t *testing.T) {
 		}
 	}
 
+	// Check the number of metrics incremented during this test (for consume mode, we expect
+	// three metrics with a single value for each label: two counts and the timing histogram since
+	// there shouldn't be any errors).
+	actualMetrics, err := testutil.GatherAndCount(
```

Comment:
I might maybe also provide some value, if we lint the metrics a bit like that:

```diff
+       lintProblems, err := testutil.GatherAndLint(
+               ti.app.Gatherer(),
+               "received_samples_total",
+               "sent_samples_total",
+               "sent_batch_duration_seconds",
+       )
```

Reply:
Will add